app/drivers/uart.h

```diff
@@ -15,9 +14,31 @@
 #define EUART_TX						(EUART_BASE + 5)
 #define EUART_RX						(EUART_BASE + 6)
 #define EUART_FLUSH_RX					(EUART_BASE + 7)
+#define EUART_INVALID_ARG				(EUART_BASE + 8)
+
+enum uart_stopbits_t {
```

Comment:
Why the _t suffix, we don't really use that, for example we have `struct usb_rx_queue_item`, no _t there. isn't _t more for types defined with typedef?